common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -59,6 +58,11 @@
     }
 
 
+    // ************* ANALYTICS *************
+
+    // Ophan pageview ID and browser ID are needed by Google Analytics, which runs sync at the bottom of the page
+    @InlineJs(ophanConfig().body, "ophanConfig.js")
```

Comment:
we also have `nonBlocking/getUserData.scala.js` down there, which you should be able to re-purpose/combine with the cookie reading code?